syncano_cli/execute/commands.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function
+
+import json
+import sys
+from ConfigParser import NoOptionError
+
+import click
+from syncano.exceptions import SyncanoDoesNotExist
+from syncano_cli.logger import get_logger
+
+from .connection import create_connection
+
+LOG = get_logger('syncano-execute')
+
+
+@click.group()
+def top_execute():
+    pass
+
+
+@top_execute.command()
+@click.option('--config', help=u'Account configuration file.')
+@click.argument('instance_name', envvar='SYNCANO_INSTANCE')
+@click.argument('script_endpoint_name')
+@click.option('--payload', help=u'Script payload in JSON format.')
+def execute(config, instance_name, script_endpoint_name, payload):
+    """
+    Execute script endpoint in given instance
+    """
+    try:
+        connection = create_connection(config)
+        instance = connection.Instance.please.get(instance_name)
+        se = instance.script_endpoints.get(instance_name, script_endpoint_name)
+        data = json.loads((payload or '').strip() or '{}')
+        response = se.run(**data)
+        if response.status == 'success':
+            print(json.dumps(response.result['stdout'], indent=4, sort_keys=True))
```

Comment:
I think You're almost there! :)

In Syncano script user can define their own custom response - then the field stdout is ignored and result is palces in response key, take a look at examples:

```
{u'response': {u'status': 200, u'content': u'{"one": 1}', u'content_type': u'application/json'}, u'stderr': u'', u'stdout': u''}
# output: "" (the print(json.dumps(....)))

{u'stderr': u'', u'stdout': u'Some string'}
# output: "Some string"

{u'stderr': u'', u'stdout': u'16'}
# output: "16"

{u'stderr': u'Traceback (most recent call last):\n  File "/app/source/main.py", line 24, in <module>\n    
2/0\nZeroDivisionError: integer division or modulo by zero', u'stdout': u''}
# output: ""

{u'response': {u'status': 200, u'content': u"{'one': 1}", u'content_type': u'applicattion/json'}, u'stderr': u'', u'stdout': u''}
# output: ""
```

Also the json.dumps for simple data types will wrap it with "": 12 -> "12" I think this is not needed here.